types/three/src/renderers/webxr/WebXRController.d.ts

```diff
@@ -1,13 +1,42 @@
 import { Group } from '../../objects/Group';
+import { Vector3 } from '../../math/Vector3';
 
 export type XRControllerEventType = XRSessionEventType | XRInputSourceEventType | 'disconnected' | 'connected';
 
+export interface XRHandJoints {
```

Comment:
Maybe this should be

```ts
export type XRHandJoints = Record<XRHandJoint, number>
```

_maybe_?

Reply:
Is there a way to make this partial later? Controllers will only cache used joints, so I previously used `Partial` with a strongly typed joint object.